website/project/model.py

```diff
@@ -609,6 +611,15 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     description = fields.StringField()
     category = fields.StringField(validate=validate_category, index=True)
 
+    # Representation of a nodes's license
+    # {
+    #  'id':  <id>,
+    #  'name': <name>,
+    #  'text': <license text>,
+    #  'properties' (optional): ['Year' (optional), 'Copyright Holders' (optional)]
```

Comment:
I think the format could be improved. Why not make properties a dict?

```python
'properties: {'year': 2015, 'copyright_holders': 'Sam, Steve'}
```

This is consistent removing "magic strings" (see prev. comment).